### PR TITLE
fix(generators): Dont set tests=false in sdl generators by default

### PR DESCRIPTION
**Pull request by dac09**
Fixes "users.scenarios" not being generated when we rebuild the fixture.

### CODE REVIEW

File: packages/cli/src/commands/generate/sdl/sdl.js [hunk 1]
```diff
@@ -242,7 +242,7 @@ export const builder = (yargs) => {
     .option('tests', {
       description: 'Generate test files',
       type: 'boolean',
-      default: false,
+      // don't give it a default value, it gets overwritten in  first few lines of the handler
```
Comment (dac09) [inline packages/cli/src/commands/generate/sdl/sdl.js:245]: This is the key change

**Comment by dac09** [comment]
I should add a test for this to prevent regression, will create another PR if it hasn’t been reviewed before I get to it!

File: packages/cli/src/commands/generate/sdl/sdl.js (hunk 1 above)
Comment (dthyresson) [reply packages/cli/src/commands/generate/sdl/sdl.js:245]: Oh, I wonder if I put `default: false,` on the wrong line. And intended it for the docs section.

**Review by dthyresson — APPROVED** [review]

**Comment by dac09** [comment]
No worries **David Thyresson** !

**Pull request merged**